src/coreComponents/events/EventManager.cpp

```diff
@@ -54,7 +54,7 @@ EventManager::EventManager( string const & name,
     setDescription( "Start simulation time for the global event loop." );
 
   registerWrapper( viewKeyStruct::maxTimeString(), &m_maxTime ).
-    setApplyDefaultValue( std::numeric_limits< real64 >::max() ).
+    setApplyDefaultValue( std::numeric_limits< integer >::max() * units::YearSeconds ).
```

Comment:
could even use a smaller number tbh.

Reply:
Perhaps we shouldn't use a number based on `std::numeric_limits< T >::max()`. While the actual value of `std::numeric_limits< real64 >::max()` in seconds is around `5.7e300 years`, this seems like a lot given the age of the universe is `13e9 years`, and the age of the earth is `4e9` years. Perhaps a nice number would be `1e6 years = 3.1536e+13  seconds`?